docs/README.md

```diff
@@ -20,16 +20,17 @@ We want to keep those docs as elaborate and easy to understand as possible.
 1. [Loading images and displaying background](programming/loading_images.md)
 1. [Handling blitted object undraw](programming/blit_undraw.md)
 1. [Blitting with masks](programming/blits_with_mask.md)
-1. Fonts
+1.  Fonts
+1. [Palettes](programming/palette.md)
 1. Optimizing blits
 1. Audio
-1. Debugging memory leaks
 1. Organizing your project
```

Comment:
And "Debugging memory leaks" chapter was removed from here. I know it's missing an article but it was a note to self made by me so that I won't forget writing about it.